Tasks/CMakeV1/package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "^2.9.3"
+    "azure-pipelines-task-lib": "^2.9.5"
```

Comment:
Nit: Run npm install in this directory to get the changes to show up in the package-lock as well. Alternately, run `node make.js build --task CMakeV1` to build this task which will have the same effect

Reply:
Thank you! Got it done.